src/components/Top.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState, useEffect } from "react";
+import {
+  Pane,
+  Heading,
+  Table,
+  TextInputField,
+  Button,
+  Spinner,
+  Link,
+  Icon,
+  Dialog
+} from "evergreen-ui";
+import { API, graphqlOperation } from "aws-amplify";
+import { listPostss } from "../graphql/queries";
+import { createPosts } from "../graphql/mutations";
+import { deletePosts } from "../graphql/mutations";
+
+function Top() {
+  const [name, setName] = useState("");
+  const [url, setUrl] = useState("");
+  const [description, setDescription] = useState("");
+  const [data, setData] = useState("");
+  const [isDialogShown, setIsDialogShown] = useState(false);
+
+  const regPosts = async () => {
+    const CreatePostsInput = {
```

Comment:
単なるオブジェクトだから先頭小文字がいいかな

```suggestion
    const createPostsInput = {
```

Reply:
対応済み（違和感はこのせいでした）